src/main/java/org/terasology/structureTemplates/components/NoConstructionAnimationComponent.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.structureTemplates.components;
+
+import org.terasology.entitySystem.Component;
+
+/**
+ * Add this component to a ST when neither the FallingBlock or Layer-by-Layer animation is required
+ */
+
+public class NoConstructionAnimationComponent implements Component {
+    public NoConstructionAnimationComponent() {}
```

Comment:
Don't need an empty constructor - that one gets generated for freeee! Since this might be confusing I took it out after merging (and tweaked the javadoc slightly for clarity)